Civi/Token/AbstractTokenSubscriber.php

```diff
@@ -158,6 +156,21 @@ public function evaluateTokens(TokenValueEvent $e) {
     }
   }
 
+  /**
+   * To handle variable tokens, override this function and return the active tokens.
+   *
+   * @param \Civi\Token\Event\TokenValueEvent $e
+   *
+   * @return mixed
+   */
+  public function getActiveTokens(TokenValueEvent $e) {
+    $messageTokens = $e->getTokenProcessor()->getMessageTokens();
+    if (!isset($messageTokens[$this->entity])) {
+      return NULL;
```

Comment:
@aydun just thinking should this return FALSE as it feels a bit more clearer

Reply:
@seamuslee001 Ok, I've changed that to FALSE if you think that's clearer.  I was following the pattern of the pre-existing `prefetch()` just below it, but either works.